vmware_horizon_tutorial.html

```diff
@@ -1,57 +1,202 @@
-<!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Curious, that's a downgrade from HTML 5 to the really old XHTML 1. I wonder why... Perhaps older version of knitr{,Bootstrap}? It should not affect the outcome, though.